include/rx/ranges.hpp

```diff
@@ -1115,19 +1115,39 @@ struct ZipRange {
     constexpr explicit ZipRange(std::tuple<Tx...>&& tuple) : inputs(tuple) {}
 
     [[nodiscard]] constexpr output_type get() const noexcept {
-        return output_type(std::forward_as_tuple(std::get<Inputs>(inputs).get()...));
+        return _get(std::index_sequence_for<Inputs...>{});
```

Comment:
Out of curiosity, what caused the original version to not work in all cases?

Reply:
If `Inputs` is `T, T`, then the line was

```cpp
return output_type(std::forward_as_tuple(std::get<T>(inputs).get(), std::get<T>(inputs).get()));
```

so std::get would not be able to tell when you want index 0 and when 1. → ["Fails to compile unless the tuple has exactly one element of that type."](https://en.cppreference.com/w/cpp/utility/tuple/get)